### PR TITLE
Opret - Profile

### PR DESCRIPTION
**Pull request by Qave**
Profile site css. + opret bruger siden

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ class RouteServiceProvider extends ServiceProvider
      */
     public const HOME = '/home';
     public const DASHBOARD = '/dashboard';
+    public const PROFILE = '/profile';
```
Comment (Noitcereon) [inline app/Providers/RouteServiceProvider.php:23]: // add this in routing?

File: routes/api.php [hunk 2]
```diff
@@ -17,3 +17,5 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
```
Comment (Noitcereon) [inline routes/api.php:20]: // remove redundant code below "Route::put("update", [DeviceController::class], "update"); "

**Review by Noitcereon — APPROVED** [review]
Looks good, but have to remove a line of code in the routes/api file.

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 3]
```diff
@@ -15,16 +15,16 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('firstname', 100)->update();
-            $table->string('surname', 100)->update();
+            $table->string('firstname', 100)->nullable()->update();
```
Comment (Lausius) [inline database/migrations/2014_10_12_000000_create_users_table.php:18]: skal firstname være nullable?

File: resources/views/profile.blade.php [hunk 4]
```diff
@@ -0,0 +1,204 @@
+@extends('layouts.login')
+@section('assets')
+    <link href="{{ asset('css/profile.css') }}" rel="stylesheet">
```
Comment (Lausius) [inline resources/views/profile.blade.php:3]: what does this do?

**Review by Benjo6 — APPROVED** [review]

**Review by Lausius — APPROVED** [review]

**Pull request merged**